src/Auth/project.json

```diff
@@ -16,7 +16,8 @@
     "Npgsql.EntityFrameworkCore.PostgreSQL": "1.0.0-rc2-release1",
     "Npgsql.EntityFrameworkCore.PostgreSQL.Design": "1.0.0-rc2-release1",
     "Npgsql": "3.1.1",
-    "OpenIddict": "1.0.0-*",
+    "OpenIddict.Core": "1.0.0-alpha2-0288",
```

Comment:
FYI, this is really old stuff. I'd recommend using the latest bits.

Note that `OpenIddict.EF` is now named `OpenIddict.EntityFramework` and that `OpenIddict` no longer references the assets/MVC/NWebsec modules by default (the `OpenIddict` package only references `OpenIddict.EntityFramework`). Personally, I'd revert this change and reference just `OpenIddict` here.

You can find more information on this thread: https://github.com/openiddict/openiddict-core/pull/127